### ISSUE TITLE
Simple PostgreSQL composite type breaks DBeaver SQL Edtior

### ISSUE THREAD

**Issue by JamesInform** [dbeaver/dbeaver#14111]
Error occurs at least in
DBeaver 21.2.0 and 21.2.1 on Mac
PostgreSQL 13 and 14

When I run the following command one by one

```
create type mytype as (f1 int8, f2 text);
create table mytable(id int8, my mytype);
insert into mytable values (1,(10,'first'));
select * from mytable;
insert into mytable values (2,(11,'second'));
```

I get the following error:
`SQL Error [42601]: ERROR: syntax error at or near "into"`

It seems as if the SQL Editor is broken.
A simple
`select 1;`
is sufficient to make everything work again. 

So it seems that querying and displaying PostgreSQL's composite type fields somehow breaks the current sql editor.

Is there a settings that I am now aware of or is this a bug?

Cheers
James

**Comment by LonwoLonwo** [comment]
Thanks for the bug report!

**Comment by LonwoLonwo** [comment]
As small workaround for now - last insert can be executed as script.

![2021-10-02 20_31_24-DBeaver Ultimate 21 2 2 - _postgres 13 3_Script-68](https://user-images.githubusercontent.com/45152336/135726660-57ae7840-dfa3-42d6-8426-9532a88c7dae.png)

**Comment by LonwoLonwo** [comment]
The problem is in the query.

![2021-10-02 20_34_09-Event](https://user-images.githubusercontent.com/45152336/135726696-c1cee36e-648e-4e2a-aa34-b8a94418f6fd.png)

Filters added for table with nested constraints.

**Comment by fvannee** [comment]
I'm running into the same issue. Please let me know when this fix will be in some EA version - I can test it.

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.2.3

**Comment by Matvey16** [comment]
Verified
